scripts/tests/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: '2'
```

Comment:
Can we instead just use the docker-compose in the root of the repo? Will start to get a bit tricky to manage several files. You can specify which compose to leverage, as well as which containers to build/start on the CMD line, as I see you're doing already.

Reply:
What I like about this is that it creates a different database for testing vs. development. That's a great situation to be in. I'm OK reusing the compose file but I'd like to keep the different test vs. dev database setup.

Reply:
Agree, but if we can just reuse one file, it's likely simpler. Thoughts on this @dpgaspar ?

Reply:
Mixed thoughts, on my local env I use 2 distinct `docker-compose` files it's easier to bring dev down or test down that way. Note that by default the script recycles the test db's so that we start from a clean env (no data), or keep them running, useful for running multiple times (reducing test dev or just dev loop times). So I propose one of the following paths forward:
- Change this docker-compose and URI to use different ports, think 6280, 5433, 3307 for examples and keep the file here (root repo is getting a little crowded)
- Change the ports (like above) and include these db's on the root `docker-compose` file, then just restart the containers that belong to the test env

I would say both have their pro's and con's, personally prefer the first. Thoughts @willbarrett @craig-rueda ?

Reply:
Option 3:

Reply:
👍

Reply:
@craig-rueda is that 👍 for the Option 3 I recommended?

Reply:
ok sounds like a plan